src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1149,6 +1153,24 @@ private String execDockerCmd(String command, int timeout) {
         return result;
     }
 
+    // Suggest a performance improvement if docker build takes too long.
+    static final long DOCKER_BUILD_SOFT_TIMEOUT = 30000;
+    private void dockerIgnoreHelpMessage(long startTime) {
```

Comment:
Method name should be a verb that describes what it does.

Reply:
👌